pkg/securitycontextconstraints/sccadmission/admission.go

```diff
@@ -46,15 +44,14 @@ func Register(plugins *admission.Plugins) {
 
 type constraint struct {
 	*admission.Handler
-	client     kubernetes.Interface
-	sccLister  securityv1listers.SecurityContextConstraintsLister
-	sccSynced  cache.InformerSynced
-	authorizer authorizer.Authorizer
+	sccLister       securityv1listers.SecurityContextConstraintsLister
+	namespaceLister corev1listers.NamespaceLister
+	listersSynced   []cache.InformerSynced
+	authorizer      authorizer.Authorizer
 }
 
 var (
 	_ = initializer.WantsAuthorizer(&constraint{})
-	_ = initializer.WantsExternalKubeClientSet(&constraint{})
```

Comment:
Are these wants ever used? Should we turn the clientset "want" into one that requires `SetExternalKubeInformerFactory` ?

Reply:
> Are these wants ever used? Should we turn the clientset "want" into one that requires SetExternalKubeInformerFactory ?

It's a compile-time nicety check.  I'm certainly willing to add this.